src/MyModal.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+
+
+
+class MyModal extends React.Component {
+
+
+  listenKeyboard(event) {
+    if (event.key === 'Escape' || event.keyCode === 27) {
+      this.props.onClose();
+    }
+  }
+
+  componentDidMount() {
+    if (this.props.onClose) {
+      window.addEventListener('keydown', this.listenKeyboard.bind(this), true);
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.props.onClose) {
+      window.removeEventListener('keydown', this.listenKeyboard.bind(this), true);
+    }
+  }
+
+  onDialogClick(event) {
+    event.stopPropagation();
+  }
+
+  onClose = e => {
+    this.props.onClose && this.props.onClose(e)
+  }
+
+
+  chooseCounty = e => { //this function does not work!
+    let playerSelection = this.Select.Option // this does not retrieve anything
```

Comment:
You should be able to get the value from your dropdown by using e.target.value. One way to get this information into your app is to define a function in your App.js which sets the state in your app. Then you pass down that function to your MyModal component which you can then use when you are changing the value. The function could look something like this:
```js
   handleChange = (e) => {
     this.setState({countyChosen: e.target.value})
   }
```
then in your select tag you can add the ```onChange``` attribute and set it equal to your handleChange prop:
```js
  <select id="county-select" onChange={this.props.handleChange} >
```
Now, whenever you change the value in your dropdown, the onChange handler will be called, which is your handleChange prop and function that you passed down from App. Then you have access in your App.js to the value that you chose in your dropdown.